Resources/public/js/models/extensions/ez-contentinfo-base.js

```diff
@@ -25,26 +25,36 @@ YUI.add('ez-contentinfo-base', function (Y) {
          * @method loadVersions
          * @param {Object} options
          * @param {Object} options.api (required) the JS REST client instance
+         * @param {Boolean} [options.preloadCreator]
          * @param {Function} callback
          */
         loadVersions: function (options, callback) {
             var versions = [],
-                contentService = options.api.getContentService();
+                contentService = options.api.getContentService(),
+                cb = function (error, response) {
+                    if (error) {
+                        callback(error, response);
+                        return;
+                    }
 
-            contentService.loadVersions(this.get('id'), function (error, response) {
-                if (error) {
-                    callback(error, response);
-                    return;
-                }
+                    Y.Array.each(response.document.VersionList.VersionItem, function (versionItemHash) {
```

Comment:
Nitpick: `response.document.VersionList.VersionItem.forEach()` is a better, native way of doing it